compose.yml

```diff
@@ -2,10 +2,15 @@ services:
   app:
     build: 
       context: .
-      target: development
+      target: runtime
+      args:
+        DEV: true
     volumes:
       - .:/app
+      - .:/tmp:rm
     tty: true
     stdin_open: true
 volumes:
   app:
+  tmp:
+
```

Comment:
What's the `tmp` volume for?

Reply:
i've added the /tmp volume because we are creating the poetry cache inside this folder. I will check if we need it

Reply:
I've removed it. We do not need it